map/traffic_manager.cpp

```diff
@@ -197,7 +196,7 @@ void TrafficManager::OnTrafficDataResponse(traffic::TrafficInfo const & info)
   m_drapeEngine->UpdateTraffic(coloring);
 
   // Update traffic colors for routing.
-  m_routingObserver.OnTrafficInfoAdded(info);
+  m_observer.OnTrafficInfoAdded(info);
```

Comment:
After discussion we decided to leave user provided call back with locked mutex but to use move semantics for passing big |info| param with traffic jam to OnTrafficInfoAdded. @ygorshenin I did it as a separate commit (08bbc66bed295673e305c77a23398710aac73f8a). Please review it carefully. I guess there's no unnecessary copying of info now.